app/containers/Admin/ReportingYear/NewReportingYearFormDialog.tsx

```diff
@@ -42,17 +23,38 @@ const NewReportingYearFormDialog: React.FunctionComponent<Props> = ({
   createReportingYear,
   existingYearKeys
 }) => {
-  const uiSchema = transformUiSchema(newReportingYearSchema.uiSchema);
-
   const handleSubmit = (e) => {
+    const beginningOfDay = {hour: 0, minute: 0, second: 0, millisecond: 0};
+    const endOfDay = {hour: 11, minute: 59, second: 59, millisecond: 999};
+    const formData = {
+      ...e.formData,
+      reportingPeriodStart: ensureFullTimestamp(
+        e.formData.reportingPeriodStart,
+        beginningOfDay
+      ),
+      reportingPeriodEnd: ensureFullTimestamp(
+        e.formData.reportingPeriodEnd,
+        endOfDay
+      ),
+      applicationOpenTime: ensureFullTimestamp(
+        e.formData.applicationOpenTime,
+        beginningOfDay
+      ),
+      applicationCloseTime: ensureFullTimestamp(
+        e.formData.applicationCloseTime,
+        endOfDay
+      ),
+      swrsDeadline: ensureFullTimestamp(e.formData.swrsDeadline, endOfDay)
+    };
```

Comment:
Dates are returned from the new DatePickerWidget as a date only, in the format `2020-11-06` without a time. In the DB it's stored as a timestamp, which we don't want to change. Here we set default dates of either 11:59pm or 12:00am depending on whether it's an "open" or "close" date.